### PR TITLE
fix: add kustomize cmd description

### PR DESCRIPTION
**Pull request by myishay**

### CODE REVIEW

File: cmd/kustomize/main.go [hunk 1]
```diff
@@ -94,7 +94,8 @@ func New(testCtx *test.TestCommandContext, kustomizeCtx *KustomizeContext) *cobr
 	testCommandFlags.AddFlags(kustomizeTestCommand)
 
 	kustomizeCommand := &cobra.Command{
-		Use: "kustomize",
+		Use:   "kustomize",
+		Short: "Render kustomize configs and run a policy check against them",
```
Comment (royhadad) [inline cmd/kustomize/main.go]: no long description?

Comment (hadar-co) [reply cmd/kustomize/main.go]: @adifay 
I recommend using similar/identical wording in all places for the same feature.
In the docs, we wrote it like this:
Run "datree kustomize test [path]" to render files of a kustomization.yaml file located in [path] and run a policy check against them.

Here I would suggest:
Render files defined in a kustomization.yaml file and run a policy check against them

Comment (myishay) [reply cmd/kustomize/main.go]: @royhadad  No need. it is only a reference to the test command

**Review by adifayer — DISMISSED** [review]

**Review by hadar-co — DISMISSED** [review]

File: cmd/kustomize/main.go [hunk 2]
```diff
@@ -94,7 +94,8 @@ func New(testCtx *test.TestCommandContext, kustomizeCtx *KustomizeContext) *cobr
 	testCommandFlags.AddFlags(kustomizeTestCommand)
 
 	kustomizeCommand := &cobra.Command{
-		Use: "kustomize",
+		Use:   "kustomize",
+		Short: "Render files defined in a kustomization.yaml file and run a policy check against them",
```
Comment (adifayer) [inline cmd/kustomize/main.go]: Render configs defined in a kustomization.yaml file and run a policy check against them

**Review by adifayer — CHANGES_REQUESTED** [review]

**Review by hadar-co — APPROVED** [review]

**Review by adifayer — APPROVED** [review]

**Pull request merged**